packages/@aws-cdk/aws-eks/lib/user-data.ts

```
@@ -13,8 +13,15 @@ export function renderAmazonLinuxUserData(cluster: Cluster, autoScalingGroup: au
 
   const extraArgs = new Array<string>();
 
-  extraArgs.push(`--apiserver-endpoint '${cluster.clusterEndpoint}'`);
-  extraArgs.push(`--b64-cluster-ca '${cluster.clusterCertificateAuthorityData}'`);
+  try {
+    const clusterEndpoint = cluster.clusterEndpoint;
+    const clusterCertificateAuthorityData =
+      cluster.clusterCertificateAuthorityData;
+    extraArgs.push(`--apiserver-endpoint '${clusterEndpoint}'`);
+    extraArgs.push(`--b64-cluster-ca '${clusterCertificateAuthorityData}'`);
+  } catch (e) {
+    // ignore
```

Explain why its ok to ignore this error